utils/opml.go

```diff
@@ -0,0 +1,35 @@
+package utils
```

Comment:
Very cool! Based on the structure of feedly.json and feedly.opml it looks like we should be able to unify the `json` and `xml` tags in these structs without having to write two different structs, which is really nice. 

In the future, we could put this into its own `opml` package (Go really likes lots of small packages) and we could have a `New(path string) (*OPML, error)` function that could help with the parsing so all we have to do is `opml.New("feeds.json")`.